styles/globals.css

```diff
@@ -22,6 +22,10 @@
   }
 }
 
+.text-stroke-1 {
+  text-shadow: -1px -1px 0 black, 1px -1px 0 black, -1px 1px 0 black, 1px 1px 0 black;
```

Comment:
```suggestion
  text-shadow: black 0 0 1em;
```

In my opinion, this looks cleaner and less stark.
What do you think?

Using `1em` means we can also use this for the author name and date without worrying about the size.

![Screenshot 2023-11-16 at 09 37 48](https://github.com/json-schema-org/website/assets/731158/735f2c34-564b-4196-9594-b3e127398634)